Sources/Apollo/MultipartFormData.swift

```diff
@@ -3,10 +3,10 @@ import Foundation
 /// A helper for building out multi-part form data for upload
 public class MultipartFormData {
   
-  enum FormDataError: Error, LocalizedError {
+  public enum FormDataError: Error, LocalizedError {
```

Comment:
If you're doing a custom implementation, would it make more sense to declare your own error type instead of making this one public?